tests/watch/expected-watcher.output

```diff
@@ -0,0 +1,23 @@
+Watching '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched' [use Control-C to terminate]
+
+Deploying '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched' due to change in '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched/packages/default/hello.js'
+Deployment status recorded in '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched/.nimbella'
+
+Deployed actions ('nim action get <actionName> --url' for URL):
+  - hello
+Deployment complete.  Resuming watch.
+
+
+Deploying '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched' due to change in '/Users/jamesthomas/code/nimbella/nimbella-cli/tests/watch/watched/packages/default/hello2/index.js'
```

Comment:
Do the tests work around `/Users/jamesthomas`?

Reply:
Good spot - nope! Will work out how to fix this...